### DISCUSSION TITLE
Repositório não pode ser clonado

### DISCUSSION THREAD

**Discussion by henri2005** [digitalinnovationone/dio-lab-open-source#50208] [Desafios DIO]
Estou tendo problemas ao dar o git clone. A taxa de recebimento dos objetos está muito baixa. Teria algo a ver com a conexão com a internet?

![image](https://github.com/user-attachments/assets/ce95122b-90e4-42a4-89c6-709b969ffda5)

Alguém pode me ajudar?

**Comment by Willian-Gabriel-18** [comment]
Estou com o mesmo problema, tento clonar, e o processo para nos 40%

**Reply by henri2005** [reply]
Então, Willian, eu consegui clonar algumas horas depois. Vi alguns vídeos sobre o problema e também revi a explicação da Elidiana sobre o projeto. Acaba que usando o git clone por SSH, o processo ocorreu perfeitamente. Continuou com um certo atraso nesses 40%, mas assim que atingiu os 50%, ele voltou a baixar o repositório rapidamente. 

Tenta clonar por SSH para ver se ele se dá certo.

**Comment by GMelloXp** [comment]
Tive o mesmo problema e a solução foi clonar por SSH. Realmente demora um pouco mais, porém foi solucionado o problema!
